examples/README.md

```diff
@@ -248,6 +248,7 @@ Example | File | Description
 `text` | [`ui/text.rs`](./ui/text.rs) | Illustrates creating and updating text
 `text_debug` | [`ui/text_debug.rs`](./ui/text_debug.rs) | An example for debugging text layout
 `ui` | [`ui/ui.rs`](./ui/ui.rs) | Illustrates various features of Bevy UI
+`ui_pipelined` | [`ui/ui_pipelined.rs`](./ui/ui.rs) | Illustrates various features of Bevy UI
```

Comment:
```suggestion
`ui_pipelined` | [`ui/ui_pipelined.rs`](./ui/ui_pipelined.rs) | Illustrates various features of Bevy UI
```

It won't work the way you want to otherwise ;)